README.md

```diff
@@ -0,0 +1,187 @@
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 WEBLOG <a name="about-project"></a>
+
+**Weblog** is social medial like ruby on rails project, it is designed for users to share thoughts and ideas in a web platform
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">HTML(.erb)</a></li>
+    <li><a href="https://rubyonrails.org/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+
+  ### Entity Relation Diagram
+  <div align="center">
+  <img src="./erd.png" alt="logo" width="640"  height="400" />
+  </div>
+  
+</details>
+
+<br />
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **User Authentication**
+- **Database connection**
+- **MVC**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+> - [Rails](https://guides.rubyonrails.org/)
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+> - IDE/Code Editor
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-prefered-folder
+  
+  git clone https://github.com/tsohleDev/O-Blogath.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd O-Blogath
+  
+  bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+  rspec spec/
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+## 👤 Tsohle Mokhemisi
+
+- GitHub: [@githubhandle](https://github.com/tsohleDev)
+- Twitter: [@twitterhandle](https://twitter.com/RealTsohle)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tsohle-mokhemisi-3687401b2/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Video Streaming**
+- [ ] **Catching**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] [OPTIONAL] In order to have a professionally documented README file, kindly ensure that a URL of the issue page is attracted in this section.

Reply:
ohh yes thanks